### ISSUE TITLE
Easy way to change CloseButtonCommand of Snackbar?

### ISSUE THREAD

**Issue by Zopenzop** [lepoco/wpfui#99]
Is there an easy way to modify the CloseButtonCommand of the WPFUI Snackbar?
Might be a rookie question, I haven't been involved with classes for too long.

**Comment by Zopenzop** [comment]
I made a local copy of the Snackbar class and modified the method to my choice, any way I could've done it better?

**Comment by pomianowski** [comment]
Hey, what effect do you want to achieve? Close only if something is finished or trigger some event after closing?

**Comment by Zopenzop** [comment]
Yes, I am using it to show progress of an ongoing task, so I want it to ask for confirmation and end the task on clicking the close button. It would be great to be able to directly customize the event.

**Comment by Zopenzop** [comment]
I just saw that the CloseButtonCommand property was added, however I can only get it and not set it.
